lib/puppet/provider/scheduled_task/win32_taskscheduler.rb

```diff
@@ -333,6 +343,17 @@ def translate_hash_to_trigger(puppet_trigger)
       self.fail "Unknown schedule type: #{puppet_trigger["schedule"].inspect}"
     end
 
+    if puppet_trigger['minutes_interval']
```

Comment:
Similar comment as https://github.com/puppetlabs/puppet/pull/3637/files#r26366163, I'd prefer to move this into the `daily` specific logic.